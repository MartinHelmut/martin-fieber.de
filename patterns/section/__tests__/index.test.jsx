import React from "react";
import renderer from "react-test-renderer";

import Section from "../index";

describe("Section", () => {
  test("should render", () => {
    const tree = renderer.create(<Section>Section content</Section>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("should render with headline", () => {
    const tree = renderer
      .create(<Section headline="Section headline">Section content</Section>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
