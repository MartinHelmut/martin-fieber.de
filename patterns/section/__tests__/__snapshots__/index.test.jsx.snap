// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section should render 1`] = `
<section
  className="section__StyledSection-sc-838b20-0 joKwZW"
>
  Section content
</section>
`;

exports[`Section should render with headline 1`] = `
<section
  className="section__StyledSection-sc-838b20-0 joKwZW"
>
  <h2
    className="headline__Headline-sc-187wqm3-0 section__SectionHeadline-sc-838b20-1 gaUVNO"
  >
    Section headline
  </h2>
  Section content
</section>
`;
